sys-cluster/galera/galera-25.3.32-r1.ebuild

```diff
@@ -1,82 +0,0 @@
-# Copyright 1999-2021 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=7
-
-PYTHON_COMPAT=( python3_{7,8,9} )
-
-inherit python-any-r1 scons-utils toolchain-funcs flag-o-matic
-
-DESCRIPTION="Synchronous multi-master replication engine that provides the wsrep API"
-HOMEPAGE="https://galeracluster.com"
-SRC_URI="https://releases.galeracluster.com/galera-3/source/galera-3-${PV}.tar.gz -> ${P}.tar.gz"
-LICENSE="GPL-2 BSD"
-
-SLOT="0"
-
-KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~x86"
```

Comment:
`galera-25.3.32-r1.ebuild` would certainly have some fixes over `galera-25.3.28-r2.ebuild`...
Better get `galera-25.3.32-r1.ebuild` stabilized and next drop `galera-25.3.28-r2.ebuild`.

Reply:
Bumped to .35